helmfile/50-applications.yaml

```diff
@@ -173,6 +173,21 @@ releases:
   - values/prometheus-alerts-wc.yaml.gotmpl
 {{ end }}
 
+- name: sc-servicemonitor
+  namespace: monitoring
+  labels:
+    app: prometheus-servicemonitor
+    prometheus: wc-reader
+  chart: ./charts/prometheus-servicemonitor
+  version: 0.1.1
+  missingFileHandler: Error
+  values:
+{{ if eq .Environment.Name "service_cluster" }}
+  # Note: we want to have alerts for WC since this is wc-reader, even if it is
+  # running in SC.
+  - values/wc-reader-servicemonitor.yaml.gotmpl
```

Comment:
Why are you adding a label for "wc-reader"? Same for the name of the values file.
These servicemonitors should be connected to sc-prometheus AFAIK.